### ISSUE TITLE
Trouble with Live Mode & Camera Post Firmware.

### ISSUE THREAD

**Issue by RoyalGreen** [fwestenberg/reolink_dev#484]
HI there,

I have two Reolink cameras:

RLC-823A  Fireware: v3.1.0.804_22011511_v1.0.0.30
E1 Outdoor Firmware: v3.1.0.804_22011506
Reolink IP Camera v0.51

I have installed the Fwestenberg integration via HACs and set both cameras to use the base stream in the Reolink app.

However since the most recent firmware upgrade to the cameras, the live view doesn't seem to work - The wheel just spins. 

Only the AUTO option seems to work and attempting to set the camera to live, will make Home Assistant to go a bit... slow/glitchy - menus don't load. This is fixed if I close the session and log back in. It only happens when i try and switch the camera to live.

Also the camera motion sensor alerts etc doesn't seem to trigger in HA.

This all used to work, so think it might be the firmware update?

Any ideas? very best wishes,

![image](https://user-images.githubusercontent.com/82901601/162687829-38484b8a-26e6-4a62-8e7e-e3a12b625289.png)

**Comment by jkili** [comment]
Have had the same problem upgrading to all 3.1.x firmware on RLC-811A and RLC-810A devices. SSL Certificate looks to be different as well on new Firmware (Chrome wont let you browse at all to them). 

Downgrading fixes errors with registering events, and live streams work again, so obviously an issue with new firmware.

**Comment by RoyalGreen** [comment]
Thank you for confirming there's a problem.

Can I check, you were only able to resolve once you downgraded the Reolink firmware?

The last firmware is feature rich on my camera, so will weigh this up.

**Comment by jkili** [comment]
No, it does work with latest firmware on 810A and 811A cameras.

Did you enable on the other streams (RSTP, etc)? I had the same problem with another camera today, and first tried replacing the certificate with a self signed cert and it didn't fix it, but I then enabled all the other network ports (RSTP, etc) and it did work. I did need to restart Home Assistant as well.
